plugins/@grouparoo/mailchimp/__tests__/integration/mailchimpImport.ts

```diff
@@ -417,9 +417,9 @@ describe("integration/runs/mailchimp-import", () => {
 
         await run.updateTotals();
         expect(run.state).toBe("complete");
-        expect(run.importsCreated).toBe(10);
+        expect(run.importsCreated).toBe(20);
         expect(run.recordsCreated).toBe(0);
-        expect(run.recordsImported).toBe(5);
+        expect(run.recordsImported).toBe(10);
```

Comment:
The mailchimp importer was always sort of wonky, and dependent on the test data that we keep seeming to delete 😢

Reply:
yeah... i was really confused when there was nothing there- same happened with snowflake, too 🤔